platform/ios/src/MGLMapAccessibilityElement.m

```diff
@@ -0,0 +1,217 @@
+#import "MGLMapAccessibilityElement.h"
+#import "MGLDistanceFormatter.h"
+#import "MGLCompassDirectionFormatter.h"
+#import "MGLFeature.h"
+#import "MGLVectorSource+MGLAdditions.h"
+
+#import "NSBundle+MGLAdditions.h"
+
+typedef CLLocationDegrees MGLLocationRadians;
+typedef CLLocationDirection MGLRadianDirection;
+typedef struct {
+    MGLLocationRadians latitude;
+    MGLLocationRadians longitude;
+} MGLRadianCoordinate2D;
+
+MGLRadianCoordinate2D MGLRadianCoordinate2DMake(MGLLocationRadians latitude, MGLLocationRadians longitude) {
```

Comment:
We already have defined this function in https://github.com/mapbox/mapbox-gl-native/blob/master/platform/darwin/src/MGLGeometry_Private.h#L25